doc/source/whatsnew/v0.21.0.txt

```diff
@@ -580,6 +580,7 @@ I/O
 - Bug in :func:`read_html` where import check fails when run in multiple threads (:issue:`16928`)
 - Bug in :func:`read_csv` where automatic delimiter detection caused a ``TypeError`` to be thrown when a bad line was encountered rather than the correct error message (:issue:`13374`)
 - Bug in ``DataFrame.to_html()`` with ``notebook=True`` where DataFrames with named indices or non-MultiIndex indices had undesired horizontal or vertical alignment for column or row labels, respectively (:issue:`16792`)
+- Bug in :func:`HDFStore.select` ValueError when doing HDFStore.Select of contiguous mixed-data table ft. VLArray (:issue:`17021`)
```

Comment:
you have an extra ValueError here.

Bug in `` :func:`HDFStore.selct` `` when reading a contiguous mixed-data .....